web-ui/src/main/resources/catalog/components/viewer/wps/WpsDirective.js

```diff
@@ -163,6 +166,15 @@
               }
             }
 
+            scope.isDateTime = function(date) {
+              return date.title.value === 'datetime';
+            };
+
+            scope.checkOutput = function (outputs) {
+              return outputs.filter(function(o) {
+                return o.reference.mimeType !== 'application/x-ogc-wms';
+            });
```

Comment:
`checkOutput` function not closed causing JS error. Fixing this in another PR targeting 4.0.x.